app/controllers/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+class UsersController < ApplicationController
+  def show
+    user_id = params[:id] || current_user.id
+    @user = User.find(user_id)
+  end
+
+  def item
```

Comment:
What is what for?  `def item`?  I couldn't tell you.  I don't even have it on my branch anymore, I must have tried to fulfill what I thought the checkpoint wanted, but I ended up deleting it.